app/services/GitRepositoryService.scala

```diff
@@ -107,24 +122,33 @@ class GitRepositoryService @Inject()(configuration: Configuration) {
   private def fetchRepositoryNames(): Seq[String] = {
     Logger.info("Fetching repositories from gitlab")
 
-    val groupIds = getProperty("gitlab.group.ids")
+    val gitlabGroupIds = getProperty("gitlab.group-ids")
+
+    val gitlabNames = gitlabGroupIds
+      .split(',')
+      .filter(_.nonEmpty)
+      .flatMap(groupId => fetchGitlabRepositoryNames(groupId))
+
+    val githubUsers = getProperty("github.users")
 
-    groupIds
+    val githubNames = githubUsers
       .split(',')
       .filter(_.nonEmpty)
-      .flatMap(groupId => fetchRepositoryNames(groupId))
+      .flatMap(groupId => fetchGithubRepositoryNames(groupId))
+
+    gitlabNames ++ githubNames
   }
 
   /**
-    * Fetch the repository names for the given groupId using gitlab api.
+    * Fetch the gitlab repository names for the given groupId using gitlab api.
     *
     * @param groupId The gitlab group id
     * @return a sequence of all the repository names
     */
-  private def fetchRepositoryNames(groupId: String): Seq[String] = {
+  private def fetchGitlabRepositoryNames(groupId: String): Seq[String] = {
 
     // gitlab api url
-    val url: String = getProperty("gitlab.url") + "/api/v4/groups/" + groupId + "/projects?per_page=1000"
+    val url: String = s"${getProperty("gitlab.url")}/api/v4/groups/$groupId/projects?per_page=1000"
```

Comment:
In practice, Gitlab usually limits the answers to 100 per page, and since there's no iteration, we're truncating the fetched repositories to the first page. You can use the [headers](https://gitlab.com/help/api/README.md#pagination-link-header) to find out whether there's a next page or not.

It's not a new issue, but the problem is also present in the GitHub implementation.